types/v56/types.go

```diff
@@ -1110,8 +1110,8 @@ type VAppLeaseSettings struct {
 // Description: Represents the ldap settings of a VMware Cloud Director organization.
 // Since: 0.9
 type OrgLdapSettingsType struct {
-	XMLName xml.Name `xml:"OrgLdapSettings"`
-	Xmlns   string   `xml:"xmlns,attr,omitempty"`
+	XMLName xml.Name `xml:"OrgLdapSettings" json:"-"`
```

Comment:
Not sure why JSON marshaler is picking untagged elements, but seems that forcing with `-` works

Reply:
Let's add a comment with a link to
https://pkg.go.dev/encoding/json
for the sake of clarity?

Reply:
Added